### DISCUSSION TITLE
[BUG] manual Weapons damage own ship

### DISCUSSION THREAD

**Discussion by Battlepixel** [tsunamayo/Starship-EVO#4028] [Suggestions]
- Starship EVO version: 21w04b

[Player.zip](https://github.com/tsunamayo/Starship-EVO/files/5895421/Player.zip)

![Starship EVO_2021_01_29_18_44_51_645](https://user-images.githubusercontent.com/43222403/106309680-c4065b00-6262-11eb-964d-c9e50e992207.jpg)

**Comment by tsunamayo** [comment]
Yes for manual weapon there is no check if I recall, it is by design. You get hit because of gimbal? Thanks

**Comment by Battlepixel** [comment]
yes it would be good to simply forbid selfharm?
